magiceightball/src/App.js

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import { useState } from "react";
+import "./styles.css";
+import { FaMagic } from 'react-icons/fa';
+
+
+
+function App() {
+    const [question, setQuestion] = useState("");
+    const [answer, setAnswer] = useState("");
+    const [errorMessage, setErrorMeessage] = useState("");
+    const [isLoading, setIsLoading] = useState(false);
+    const [imageSrc, setImageSrc] =useState("");
+
+    const handleChange = (e) => {
+      setQuestion(e.target.value);
+    }
+    
+    const handleSubmit = (e) => {
+      e.preventDefault();
+      if(question.slice(-1) !== '?' || question === ""){
+        setErrorMeessage("You didn't ask a question!");
+        setIsLoading(false);
+        return;
+      } else {
+        setErrorMeessage("");
+        setIsLoading(true);
+        setAnswer("");
+        fetch("https://yesno.wtf/api")
+          .then((response) => response.json())
+          .then((data) => {
+            setAnswer(data.answer);
+            setImageSrc(data.image);
+            setIsLoading(false);
+            console.log(data);
+          }).catch((error) => {
+            setErrorMeessage("Error: " + error.message);
+            setIsLoading(false);
+            console.log(error);
+          }
+          );
+      }
+    }
+
+  return (
+    <div className="App">
+      <h1 className="pTitle pAnimation">Magic 8 Ball <FaMagic></FaMagic></h1> 
+      <h2 className="pQuestion">Ask your question.</h2>
+      <form onSubmit={handleSubmit}>
+        <input className="input" type="text" onChange={handleChange} placeholder=""/>
+        <br></br>
+        <button className="" type="submit">Ask!</button>
+        {errorMessage ? <p>{errorMessage}</p> : null}
+        {isLoading ? <p>Loading...</p> : null}
+        {answer ? 
```

Comment:
separate to another component like it's said in `7) Make asked question, answer and answer image as new component in different file.`